ansible/roles/test/files/ptftests/py3/copp_tests.py

```diff
@@ -492,6 +524,38 @@ def contruct_packet(self, port_number):
 
         return packet
 
+    def check_constraints(self, send_count, recv_count, time_delta_ms, rx_pps):
+        self.log("")
```

Comment:
@prabhataravind  log a new line here?

Reply:
@ZhaohuiS, I retained the same formatting style used in check_constraints() function in policyTest class in line 257 above.